VERSIONING.md

```diff
@@ -73,28 +73,27 @@ a:
 - Docs: :book: (`:book:`)
 - Infra/Tests/Other: :seedling: (`:seedling:`)
 
-You can also use the equivalent emoji directly, since GitHub doesn't
-render the `:xyz:` aliases in PR titles.
-
 Individual commits should not be tagged separately, but will generally be
 assumed to match the PR. For instance, if you have a bug fix in with
 a breaking change, it's generally encouraged to submit the bug fix
 separately, but if you must put them in one PR, mark the commit
 separately.
 
+All changes must be code reviewed. Coding conventions and standards are explained in the official [developer
+docs](https://git.k8s.io/community/contributors/devel). Expect reviewers to request that you
+avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments) in your PRs.
+
 ### Commands and Workflow
 
 Cluster API Provider Metal3 follows the standard Kubernetes workflow: any PR
-needs `lgtm` and `approved` labels, PRs authors must have signed the CNCF CLA,
-and PRs must pass the tests before being merged.  See [the contributor
-docs](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#the-testing-and-merge-workflow)
-for more info.
+needs `lgtm` and `approved` labels, and PRs must pass the tests before being merged.
+See [the contributor docs](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#the-testing-and-merge-workflow) for more info.
```

Comment:
Shall we keep CNCF CLA and ask for it from authors rather ?

Reply:
Before change, it says `PRs authors must have signed the CNCF CLA,` which makes me think we have a mechanism to check if it is not the case, so I removed it to make it clear. Can you elaborate more what you mean by ask from authors?

Reply:
to have CNCF CLA

Reply:
We have to check if we can check existence of CNCF CLA in prow, for now I am ok to remove it and we can check it later

Reply:
Sure, I will look into putting CNCF CLA check in prow for this repo at least.